.github/workflows/auto-fix-and-pr.yml

```
@@ -0,0 +1,44 @@
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run auto-fixers (frontend + backend)
+        run: |
+          npm run lint:fix || true
+          npm run lint:fix:workflow-studio || true
+          npm run type-check || true
+
+      - name: Create Pull Request with fixes
+        uses: peter-evans/create-pull-request@v4
```

actionlint (1.7.8)

the runner of "peter-evans/create-pull-request@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)